components/admin_console/system_users/system_users_list.jsx

```diff
@@ -269,6 +270,7 @@ export default class SystemUsersList extends React.Component {
                     page={this.state.page}
                     term={this.props.term}
                     onTermChange={this.props.onTermChange}
+                    rowComponentType={UserListRowWithError}
```

Comment:
If this prop isn't set, the standard `UserListRow` component is used instead, so this change doesn't break any other uses of `UserList`